### DISCUSSION TITLE
Checking a resource where against a attribute that is an array

### DISCUSSION THREAD

**Discussion by DavePCoder** [cerbos/cerbos#1969] [Q&A] [answered]
Forgive the newbie question to the world of Cerbos. The examples have helped me do a lot of the work i needed, but i'm struggling with how to tackle the following scenario:
- I have a 'task' resource
- In my app as an admin i can assign multiple users to a task
- Those users i task can then edit the estimated completion date for the task
- So ultimately the principal is the current user, the resource is a task, the action is edit_forecast

How how would i define the resource attributes?  I effectively have an attribute that is a array or collection that is UsersOnTask, and i want to EFFECT_ALLOW edit_forecast if the principle.id is any id in the UsersOnTask (array) attribute.  However, as far as i can tell i cannot have an array as an attribute, so how would i achieve this?

**Comment by charithe** [comment, marked as answer]
Hi. You can absolutely have an array as an attribute. You can see an example here where `teams` is a list of strings: https://docs.cerbos.dev/cerbos/latest/policies/conditions#_lists_and_maps

**Comment by charithe** [comment]
This demo uses multiple array attributes as well: https://github.com/cerbos/demo-multitenant-saas
